src/components/Sidebar/Sidebar.js

```diff
@@ -127,6 +127,19 @@ class Sidebar extends PureComponent<Props, State> {
             <Wrapper offset={`${sidebarOffsetPercentage}%`}>
               <ScrollbarOnlyVertical
                 autoHide
+                renderThumbVertical={({ style, ...props }) => (
+                  // Info: Styled-components not working here yet
+                  //       --> PR 286 @ react-custom-scrollbars will fix this
+                  <div
+                    {...props}
+                    style={{
+                      ...style,
+                      background: COLORS.transparentWhite[700],
```

Comment:
@melanieseltzer Could you please check if the scrollbar looks better if we're having the default background (for the automatic scrollbar setting)? Just comment line 137. If it's looking better we could change the look only for Mac.

Reply:
No change for me commenting out line 137.

I think let's merge this since we're already using `react-custom-scrollbars` and then create another issue logging this bug/see if another library would work better.